### ISSUE TITLE
Checkbox is not working?

### ISSUE THREAD

**Issue by ahosker** [livewire/flux#474]
I want to  replace this:
```blade
    <div>
        <input type="checkbox" wire:model.live="is_first_time_buyer" class="">
        <label for="is_first_time_buyer">Are you a First Time Buyer?</label>
    </div>
```
with this:

```blade
<flux:checkbox wire:model.live="is_first_time_buyer" label="You are a first time buyer" />
```

However, the true/false values are not being reflected in the backend.

In the livewire component, I have:
```php
    #[Url(keep: false, history: true, except: false)]
    #[Validate('required|boolean')]
    public bool $is_first_time_buyer = false;
```

Then a simple math formula, but when I use flux checkbox its not reflecting the change at all.

**Comment by fabioserembe** [comment]
Hey.. same issue a couple of days ago :) [#467](https://github.com/livewire/flux/issues/467)
Resolved by updating Flux to v1.0.14 and Livewire to v3.5.12

**Comment by calebporzio** [comment]
Thanks @fabioserembe - yes, this should be all set in the latest versions:

![Image](https://github.com/user-attachments/assets/b18cafdc-f66a-42b1-94aa-3766919f6e91)
